references/cli/portforward.go

```diff
@@ -293,25 +293,28 @@ func (o *VelaPortForwardOptions) Complete() error {
 		var found bool
 		_, configs := appfile.GetApplicationSettings(o.App, compName)
 		for k, v := range configs {
-			if k == "port" {
```

Comment:
please keep the port here

Reply:
The port in webservice not generate service. we need port-forward it?